draft-ietf-cdni-uri-signing.xml

```diff
@@ -608,6 +635,33 @@
           <section anchor="uri_container_forms_hash" title="URI Hash Container (uri-hash:)">
               <t>Prefixed with 'uri-hash:', this string is a URL Segment form (<xref target="RFC6920"/> Section 5) of the URI.</t>
           </section>
+
+          <section anchor="uri_container_forms_b" title="URI Base Container (b:)">
+              <t>This string is simply 'b:', with no additional
+              information. This form indicates that the URI MUST
+              match the Base claim of the JWT. A URI matches iff:
+              <list style="symbols">
+                  <t>If the scheme is present in the Base claim, it is,
+                  according to a case insensitive comparison, the same
+                  as the scheme of the URI; and</t>
+                  <t>If the host is present in the Base claim, it is,
+                  according to a case insensitive comparison, the same
+                  as the host of the URI; and</t>
+                  <t>If the port is present in the Base claim, it matches
+                  the port on which the request is received; and</t>
+		  <t>The path in the Base claim is either empty following a
+		  host or schema, a single slash ('/'), or each of its
+		  components match, using a case sensitive comparison, each of
+		  the path components, in order, that start the URI.</t>
+              </list></t>
+
+	      <t>If the signed JWT does not contain a CDNI Base claim, or its
+	      Base claim is empty, the request MUST be rejected.</t>
```

Comment:
In another section you mentioned that this claim woudn't be grounds for accepting or rejecting. It looks like you are also adding a new URI container type here though. This usage isn't clear to me from the text.

Reply:
To be clear: the Base claim is not a basis for accepting or rejecting. Rather, the `b:` container relies on information in the base claim in order to accept or reject.

A token provider may wish to use the base claim to document the path for cookie propagation, but prefer to use a regex or simple container for validation.

Reply:
So you intend to have people pass a `cdnib` claim plus a `b:` in `cdniuc` plus a `uri-regex` in a second `cdniuc`?